### PR TITLE
Bump request from 2.33.0 to 2.88.2

### PR DESCRIPTION
**Pull request by ShaneKilkelly**
Branched from https://github.com/overleaf/track-changes/pull/76 to make CI pass

Closes https://github.com/overleaf/track-changes/pull/76

----

Bumps [request](https://github.com/request/request) from 2.33.0 to 2.88.2.
- [Release notes](https://github.com/request/request/releases)
- [Changelog](https://github.com/request/request/blob/master/CHANGELOG.md)
- [Commits](https://github.com/request/request/commits)

Signed-off-by: dependabot[bot] <support@github.com>

#### Manual Testing Performed

- [ ]
- [ ]

#### Deployment Checklist

- [ ] Update documentation not included in the PR (if any)
- [ ]

### CODE REVIEW

**Comment by ShaneKilkelly** [comment]
Tests pass locally. The build triggers seem to still be associated with one of the previous builds on the old dependabot branch name. The latter trigger is passing.

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "mongojs": "2.4.0",
     "redis": "~0.10.1",
     "redis-sharelatex": "^1.0.8",
-    "request": "~2.33.0",
+    "request": "~2.88.2",
     "requestretry": "^1.12.0",
```
Comment (das7pad) [inline package.json]: We should consider upgrading `requestretry` too, see https://github.com/overleaf/document-updater/pull/121#discussion_r380380669 for details.

Comment (ShaneKilkelly) [reply package.json]: Good catch, updated :+1:

**Comment by ShaneKilkelly** [comment]
I've opened a new issue to address the build failures on dependabot PRs: https://github.com/overleaf/issues/issues/2793

**Pull request merged**